DigitalLearningSolutions.Data/Models/SearchSortFilterPaginate/SearchSortFilterAndPaginateOptions.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace DigitalLearningSolutions.Data.Models.SearchSortFilterPaginate
+{
+    public class SearchSortFilterAndPaginateOptions

```

Comment:
It would be nice to have an xml doc on this class to explain how to turn off sort / search / filter for a page. And what happens if paginationOptions is null.